deploy/static/provider/oci/kustomization.yaml

```diff
@@ -4,7 +4,7 @@
 # ```
 # namespace: ingress-nginx
 # bases:
-#   - github.com/kubernetes/ingress-nginx/tree/main/deploy/static/provider/cloud
+#   - github.com/kubernetes/ingress-nginx/tree/main/deploy/static/provider/oci
```

Comment:
so oci is confusing, can we spell it out like oraclecloud or just oracle?

Reply:
Updated